dataproc_jupyter_plugin/controllers/executor.py

```diff
@@ -12,24 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 import json
+from dataproc_jupyter_plugin import credentials
```

Comment:
Wrong import order (see https://peps.python.org/pep-0008/#imports).

The imports should go as follows:

1. Standard library imports
2. Third-party imports
3. Same package imports

... so the credentials import (from this same package) should be *after* the tornado import (from a third-party package).